src/release/breaking-changes/rendering-changes.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Notable Rendering and Layout Changes after v3.7
+description: Non-API related breaking changes made after Flutter v3.7.
+---
+
+## Changes
+
+This section lists the notable non-API breaking changes.
+
+### (Only Affects Tests) `FlutterTest` is now the default test font
+
+The `FlutterTest` font replaced `Ahem` as the default font in tests: when 
+`fontFamily` isn't specfied, or the font families specified are not registered, 
+tests use the `FlutterTest` font to render text. The `Ahem` font is still 
+available in tests if specified as the `fontFamily` to use.
+
+The `FlutterTest` font produces more precise font and glyph metrics than `Ahem`, 
+and the metrics are generally font-engine agnostic. See the [Flutter Test Fonts][] 
+wiki page for more details about the test font.
+
+**Differences**
+
+The `FlutterTest` font looks almost identical to the old default `Ahem`: the
+glyph for most characters is a box that fills the em square. The notable
+differences between the `FlutterTest` font and `Ahem` font are:
+
+**1. Different Baseline Location**
+
+The `FlutterTest` font's ascent and descent are 0.75 em and 0.25 em, while
+`Ahem`'s are 0.8 em and 0.2 em respectively.
+
+In the example golden image change below, the white blocks are text rendered
+using `Ahem` and `FlutterTest`. The second character is taller in the new font
+since it has a larger descent.
+| Before (`Ahem`) | After | Animated Diff |
+| :---: | :---: | :---: |
+| ![before](assets/material.ink_sparkle.bottom_right.0_masterImage.png) | ![after](assets/material.ink_sparkle.bottom_right.0_testImage.png) | ![baseline_animated](assets/baseline.gif) |
```

Comment:
I'm not sure what the best practices are for adding and managing image assets in breaking change notes so I created an assets folder.

Reply:
That's fine.  I probably would have called it images, but assets is ok.

Reply:
Should I change the name of the markdown file to something like `rendering-changes-v-3-7.md`?

Reply:
It's fine